build/ndm-operator/Dockerfile.in

```diff
@@ -0,0 +1,14 @@
+#
+# This Dockerfile builds node-disk-operator
+# 
+
+#Set the base image
+FROM @BASEIMAGE@
+
+ARG ARCH
+
+RUN apt update
+
+ADD ndo /usr/local/bin/ndo
+
+ENTRYPOINT ["./ndo"]
```

Comment:
You have to use it like `ENTRYPOINT ["/usr/local/bin/ndo"]`, else it will give no such file error.